### PR TITLE
[L2B-5559] Add last used component for verifiers

### PR DESCRIPTION
**Pull request by maciekop-l2b**
Resolves L2B-5559

### CODE REVIEW

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/frontend/src/pages/project/zk-catalog/view/Verifiers.tsx [hunk 1]
```diff
@@ -76,7 +83,13 @@ export function Verifiers(props: Props) {
...
               <div>
                 <p className="text-gray-500 dark:text-gray-50 text-xs mb-2 font-medium">
-                  Description
+                  Last used
+                </p>
+                <LastUsedCell days={item.lastUsedDaysAgo} />
+              </div>
+              <div>
+                <p className="text-gray-500 dark:text-gray-50 text-xs mb-2 font-medium">
+                  Descriptions
```
[the remark this answers is not in the document]
Comment (maciekop-l2b) [reply packages/frontend/src/pages/project/zk-catalog/view/Verifiers.tsx:92]: Fixed

File: packages/frontend/src/build/api/fetchVerifiersApi.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  if (backend.mock) {
+    return getVerifiersApiResponse()
+  }
+
+  const url = backend.apiUrl + '/api/verifiers'
+  const json = await http.fetchJson(url)
+  return VerifiersApiResponse.parse(json)
+}
+
+function getVerifiersApiResponse(): VerifiersApiResponse {
```
[the remark this answers is not in the document]
Comment (maciekop-l2b) [reply packages/frontend/src/build/api/fetchVerifiersApi.ts:23]: Fixed

File: packages/frontend/src/pages/project/zk-catalog/props/getProps.ts [hunk 3]
```diff
@@ -31,33 +33,30 @@ export function getProps(
...
-  assert(project.stateValidation?.proofVerification, 'Invalid project')
+  if (project.type === 'zk-catalog') {
+    return baseDetails
+  }
 
   return {
-    title: project.display.name,
-    icon: `/icons/${project.display.slug}.png`,
+    ...baseDetails,
     linkToMainProjectDetails: `/scaling/projects/${project.display.slug}`,
```
[the remark this answers is not in the document]
Comment (maciekop-l2b) [reply packages/frontend/src/pages/project/zk-catalog/props/getProps.ts:60]: done

File: packages/frontend/src/pages/project/zk-catalog/props/getProps.ts [hunk 4]
```diff
@@ -42,21 +42,15 @@ function getZkCatalogProjectDetails(
...
 
-  const baseDetails = {
+  return {
     title: project.display.name,
     icon: `/icons/${project.display.slug}.png`,
-    linkToMainProjectDetails: undefined,
+    linkToMainProjectDetails:
+      project.type === 'zk-catalog'
+        ? `/scaling/projects/${project.display.slug}`
+        : undefined,
```
Comment (torztomasz) [inline packages/frontend/src/pages/project/zk-catalog/props/getProps.ts:51]:
```suggestion
    linkToMainProjectDetails:
      project.type === 'zk-catalog'
        ? undefined
        : `/scaling/projects/${project.display.slug}`,
```

**Review by torztomasz — APPROVED** [review]

**Pull request merged**